File: Ejemplo4.java
public class Ejemplo4 {
    static class Persona{
        private int idPersona;
        private String nombre;
        private String apellido;
        public Persona(){}
        public Persona(int id, String n, String a){
            this.idPersona = id;
            this.nombre = n;
            this.apellido = a;
        }
        public int getIdPersona(){
            return this.idPersona;
        }
        public String getNombre(){
            return this.nombre;
        }
        public String getApellido(){
            return this.apellido;
        }
    }
    public static void main(String[] args) {
        Persona p = new Persona(1, "Juan", "Perez");
        System.out.println(p.getNombre());

    }
}

File: Ejemplo4.py
class Persona:
    def __init__(self, id, nom, ape):
        self.__idPersona = id
        self.__nombre = nom
        self.__apellido = ape

    def getIdPersona(self):
        return self.__idPersona
    def getNombre(self):
        return self.__nombre
    def getApellido(self):
        return self.__apellido
    
ob = Persona(1, "Juan", "Perez")
print(type(ob))